### PR TITLE
156005300 fastlane deploy to slack

### PR DESCRIPTION
**Pull request by simiyu1**
#### What does this PR do?
 - Integrate fastlane deployment to slack

#### Description of Task to be completed?
 - call slack webhooks from fastfile
 - create lane for slack in fastfile
 
#### How should this be manually tested?
 - Clone this repository to your machine, select app and run the application on an emulator or device.
 - on your terminal run nano ~/.zshrc to edit the file
 -  then in the file write 
```
     export LC_ALL=en_US.UTF-8
     export LANG=en_US.UTF-8
     export SLACK_URL="...Replace with workspaceURL..."
     export SLACK_API_TOKEN="...Replace with actual token..."
     export SLACK_CHANNEL="...Replace with actual channel..."
```
- Save and close nano and then run command `bundle exec fastlane slackbuild` 
- It should generate an APK into the slack channel indicated above.

#### What are the relevant pivotal tracker stories?
 - [#156005300](https://www.pivotaltracker.com/story/show/156005300)

#### Screenshots
image

### CODE REVIEW

File: .zshrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export LANG=en_US.UTF-8
```
Comment (Rwothoromo) [inline .zshrc:1]: @simiyu1 This file is not necessary on GitHub, please remove it. Also, it exposes private data.

**Review by Rwothoromo — COMMENTED** [review]
@simiyu1 Good work. However, remove files to do with `.zshrc` and `Fastlane` that contain sensitive data. Instead, write instructions on how one should go about some of those steps.
Also remove the files that would be auto-generated for someone setting up on another local machine.

File: .zshrc (hunk 1 above)
Comment (simiyu1) [reply .zshrc:1]: On it!

Comment (simiyu1) [reply .zshrc:1]: Resolved

**Pull request merged**